src/esp/assets/ResourceManager.h

```diff
@@ -1196,6 +1209,7 @@ class ResourceManager {
    * @brief The mesh data for loaded assets.
    */
   std::map<int, std::shared_ptr<BaseMesh>> meshes_;
+  std::pair<int, int> drawableCountAndNumFaces_{0, 0};
```

Comment:
Should we use Corrade Pair here?

Reply:
StringView for that list of strings might make sense too, if they will always be static.

But neither seems too perf-critical to me, so I don't mind either way.